core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/scatterplot/ScatterplotOpDesc.scala

```diff
@@ -58,15 +64,15 @@ class ScatterplotOpDesc extends VisualizationOperator with PythonOperatorDescrip
     assert(xColumn.nonEmpty && yColumn.nonEmpty)
     s"""
        |        # drops rows with missing values pertaining to relevant columns
-       |        table.dropna(subset=['$xColumn', '$yColumn'], inplace = True)
+       |        table.dropna(subset=['$xColumn', '$yColumn', '$colorColumn'], inplace = True)
```

Comment:
Does a user have to provide color column?

Reply:
it's optional.